labs/dxp/7.x/installation-and-upgrades/02-setting-up-liferay-dxp/installing-and-managing-apps/installing-apps-via-the-file-system.md

```diff
@@ -0,0 +1,27 @@
+# Installing Apps via the File System
+
+Installing apps via the file system is the best method for most administrators. Installing via the file system with a system restart is the optimal way to install apps in production-grade environments, as it avoids potential issues that may arise when deploying apps with auto deployment.
```

Comment:
Saying that installing via the file system is best for most administrators doesn't give any reason. One reason we could give is that it's natural to work with the file system once you've downloaded an app as a file. 

Also, saying that it's for "administrators" makes me wonder if this approach is applicable to me (the reader). Do I consider myself an "administrator"? If not, should I skip this approach? Let's let the reader decide whether they like the approach and leave roles out of it. :-D

Here's one way to introduce this topic:

After [downloading an app](TODO) as an LPKG file from the  [Marketplace website], (https://web.liferay.com/marketplace), it's natural to copy it to a local installation. Or if you're working with DXP nodes in a cluster, you can copy the file to each node host and install it. 

Copying the file to your DXP installation while DXP is shutdown is the optimal way to install apps in production-grade environments, as it avoids potential issues that may arise from auto deploying apps. See [Installing Apps via the UI](TODO) for details.

Reply:
@Alec-Shay - I included module JAR and plugin WAR with app LPKG files as they can be deployed the same way as LPKG files.

Reply:
Oh, never mind; I see all three are mentioned in step 2. :-D

Reply:
@jhinkey The wording could certainly be improved; I sort of struggled to come up with a good starting sentence (the first bit always seems to be the hardest) and that was the best I came up with, which I agree kind of doesn't really help the reader much.

That said, as far as:

> Let's let the reader decide whether they like the approach and leave roles out of it.

I think part of what we are trying to do is help direct the reader to not leave them with an ambiguous decision of which to use, aren't we? So at least if I'm understanding correctly, we should avoid just presenting decisions as "you could do it this way, if you want, if you like it, (etc.)."... even if, realistically, that is sort of what it comes down to.